tests/test_wheel.py

```diff
@@ -34,6 +35,14 @@
 
 if platform.system() == 'Linux':
     PLATFORM_TAG = f'linux_{platform.machine()}'
+    SHARED_LIB_EXT = 'so'
+elif platform.system() == 'Darwin':
+    PLATFORM_TAG = '_'.join((
```

Comment:
I think this needs to use the same logic as `Project.platform_tag`

Reply:
Yeah, that's probably for the best.